app-modules/portal/src/Http/Controllers/KnowledgeManagement/KnowledgeManagementPortalArticleController.php

```diff
@@ -48,8 +49,14 @@ class KnowledgeManagementPortalArticleController extends Controller
 {
     public function show(KnowledgeBaseCategory $category, KnowledgeBaseArticle $article): JsonResponse
     {
+        $portalViewCountFlag = Feature::active('portal_view_count');
+
         $articleUpdatedAt = $article->updated_at->setTimezone(app(DisplaySettings::class)->timezone);
 
+        if ($portalViewCountFlag) {
+            $article->update(['portal_view_count' => $article->portal_view_count + 1]);
```

Comment:
We should use `increment` here instead as it is more performant and is an atomic operation: https://laravel.com/docs/11.x/queries#increment-and-decrement

Reply:
Done